### PR TITLE
Make the command-line-args dialog better

### PR DESCRIPTION
**Pull request by zander**
Instead of using a fixed-width font in a label, which virtually
guarantees a horizontal scrollbar, use a proper text-document
that can re-layout based on user input.

### CODE REVIEW

**Comment by fanquake** [comment]
Screenshots comparing master & this pull. Note that you've removed the version number from the page.
Master
screen shot
screen shot 1

Pull
screen shot 3
screen shot 4

**Comment by luke-jr** [comment]
I like the idea of spacing between options, but maybe only half a line?

**Comment by zander** [comment]
Re-added the version number; made the middle lines bold and made the cell-padding less.
commandlineoptions

**Comment by laanwj** [comment]
Certainly looks better.
Nit: it's still using a large fixed-width font here. Looks better after removing

    ui->helpMessage->setFont(GUIUtil::bitcoinAddressFont());

**Comment by Diapolo** [comment]
Perhaps we should also remove that graphics to the left then!?

**Comment by laanwj** [comment]
Hey, that's one of the only leftovers of Satoshi!
Anyhow, that doesn't need to be in this pull.

**Comment by jonasschnelli** [comment]
I can take care about the graphic on the left. Needs definitively a replacement (even if its chosen by Satoshi).

**Comment by jonasschnelli** [comment]
Do we need a graphic on this dialogue?

**Comment by laanwj** [comment]
No.

File: src/qt/forms/helpmessagedialog.ui [hunk 1]
```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>800</width>
-    <height>400</height>
+    <width>585</width>
```
Comment (Diapolo) [inline src/qt/forms/helpmessagedialog.ui]: Were this resizes intended?

File: src/qt/forms/helpmessagedialog.ui [hunk 2]
```diff
@@ -47,19 +54,22 @@
          <rect>
           <x>0</x>
           <y>0</y>
-          <width>659</width>
-          <height>348</height>
+          <width>447</width>
```
Comment (Diapolo) [inline src/qt/forms/helpmessagedialog.ui]: Same here...

File: src/qt/forms/helpmessagedialog.ui (hunk 1 above)
Comment (zander) [inline src/qt/forms/helpmessagedialog.ui]: yes. Much less content is shown, especially on the 'about' dialog this resize makes sense.

**Comment by jonasschnelli** [comment]
Sorry to comment post-merge here.
On windows this looks pretty compact:
bildschirmfoto 2015-01-09 um 20 08 02

On OSX it's also a bit tiny:
bildschirmfoto 2015-01-09 um 20 09 28 1

Is there a reason why this is much smaller then the main window?

**Pull request merged**